create tweet display methods

MyskYko opened the pull request:
ツイートの表示まで(更新はされない)
## やったこと
 - ツイート作成時にidが重ならないように
 - TweetCellのpropertiesをfileprivateにし、setLabel(function)で設定する
 - TimeLineViewModelを作り、initialでVariableにTweetの配列を入れる
 - tableViewをrxで表示するようにした
 - DateをStringmにするfunctionを作った

## 関連イシュー
#5 
## 注意点

## 特にレビューして欲しいポイント

![simulator screen shot 2017 04 27 20 44 26](https://cloud.githubusercontent.com/assets/18373300/25482349/573759a4-2b8c-11e7-8f67-ea432e77969b.png)

MyskYko committed:
create tweet display methods

yukitoto commented on Miyatter/MakeTweetViewModel.swift:
```diff
@@ -31,6 +31,9 @@ final class MakeTweetViewModel {
                 try! realm.write() {
                     let tweet = Tweet()
                     tweet.content = text
+                    if let lastTweet = realm.objects(Tweet.self).last {
```
idでソートしないと必ず最大のものが来る保証がないからコンフリクトの可能性あり

yukitoto commented on Miyatter/TimeLineViewController.swift:
```diff
@@ -45,6 +45,16 @@ class TimeLineViewController: UIViewController {
     // MARK: - Life Cycle Events -
     
+    init(viewModel: TimeLineViewModel) {
+        self.viewModel = viewModel
```
good

yukitoto commented on Miyatter/TimeLineViewController.swift:
```diff
@@ -97,24 +107,19 @@ class TimeLineViewController: UIViewController {
-        cell.contentLabel.text = "ツイートの内容"
-        cell.commentCountLabel.text = "コメント数:0"
-        return cell
+    fileprivate func stringFromDate(date: Date) -> String {
```
これはDateのExtensionにしよう。ViewControllerの責務とは少し離れる気がするので

yukitoto commented on Miyatter/TimeLineViewModel.swift:
```diff
@@ -0,0 +1,26 @@
+    
+    // MARK: - Properties -
+    
+    var tweetVariable: Variable<Results<Tweet> >
```
空白発見

yukitoto commented on Miyatter/TweetCell.swift:
```diff
@@ -78,5 +78,14 @@ class TweetCell: UITableViewCell {
+    // MARK: - Set Labels -
+    
+    func setLabels(date: String, content: String, commentCount: String) {
+        dateLabel.text = date
```
func update(post: Post)

yukitoto replied:
`func update(byt: Post)`
`func update(comment: Comment)`

`cell.update(post: post)`

MyskYko committed:
Modified
 - date to string func is now extension of Date
 - in TweetCell setLabels is now update(tweet:Tweet)

MyskYko commented:
訂正
## やったこと
 - StringにするfuncをDateのextensionにしました
 - update(tweet: Tweet)にしました

## 関連イシュー
#5 
## 注意点

## 特にレビューして欲しいポイント

MyskYko committed:
modified
 - id set func is now in model

MyskYko commented:
##やったこと
 - idのインクリメント処理をTweetモデルに移植
